packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -1,6 +1,15 @@
-import { Show } from 'solid-js';
+import { ParentProps, Show } from 'solid-js';
 import { InboxNotification } from '../../../types';
-import { useStyle } from '../../helpers';
+import { formatToRelativeTime, useStyle } from '../../helpers';
+import { Archive, ReadAll, Unarchive } from '../../icons';
+import { Button } from '../primitives';
+
+type NotificationBodyProps = ParentProps;
+const NotificationBody = (props: NotificationBodyProps) => {
+  const style = useStyle();
+
+  return <p class={style('notificationBody')}>{props.children}</p>;
```

Comment:
✏ **suggestion:** please talk to Nikolay about the ellipsis for the body, I remember that we agreed on max three lines, but please confirm that